mypy/checker.py

```diff
@@ -5271,10 +5271,15 @@ def format_expr_type() -> str:
             else:
                 return f"Expression has type {typ}"
 
+        def get_expr_name() -> str:
+            if hasattr(expr, "name"):
+                return f'"{str(expr.name)}"'
```

Comment:
nit: expr.name is already a str, so can just use `f'"{expr.name}"''`

Reply:
I added this in since the type wasn't clear to mypy. However, the change above fixed this :)